cdap-app-fabric/src/main/java/co/cask/cdap/adapter/DataType.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.adapter;
+
+/**
+ * Defines the type of data used in source and sink
+ */
+public enum DataType {
```

Comment:
The thinking behind not re-using that is - The sources and sinks for the adapters might be different than the types specified in ```co.cask.cdap.app.services.Data```. For instance, we could have in future adapters that read from Kafka and that might not be a good candidate for what is intended for ```co.cask.cdap.app.services.Data```.